pkgs/shells/fish/plugins/fifc.nix

```diff
@@ -0,0 +1,23 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+}:
+buildFishPlugin rec {
+  pname = "fifc";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gazorby";
+    repo = "fifc";
+    rev = "v${version}";
+    sha256 = "sha256-p5E4Mx6j8hcM1bDbeftikyhfHxQ+qPDanuM1wNqGm6E=";
+  };
+
+  meta = with lib; {
+    description = "fifc brings fzf powers on top of fish completion engine and allows customizable completion rules";
```

Comment:
```suggestion
    description = "Fzf powers on top of fish completion engine and allows customizable completion rules";
```
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.

https://nixos.org/manual/nixpkgs/stable/#var-meta-description

How about citing `Configurable fzf completions for fish shell`?